packages/pyright-internal/src/localization/localize.ts

```diff
@@ -186,27 +164,15 @@ function loadStringsForLocale(locale: string): StringLookupMap {
     return {};
 }
 
-function loadStringsFromJsonFile(locale: string): StringLookupMap | undefined {
+async function loadStringsFromJsonFile(locale: string): Promise<StringLookupMap | undefined> {
+    let result: StringLookupMap | undefined = undefined;
     switch (locale) {
-        case 'de':
-            return deStrings;
-        case 'en-us':
-            return messageStyle === 'simplified' ? mergeStrings(enUsStrings, enUsSimplified) : enUsStrings;
-        case 'es':
-            return esStrings;
+        case 'en':
+            return (await import('./simplified.nls.en-us.json')).default;
```

Comment:
This has removed the per-locale loading of non-simplified strings

As there are no translations in Pyright (all files are empty) this didn't amount to much.

I think we should just accept that this is an area where we've made a significant change and will have to maintain over time. The key thing is to maintain the default strings for the sake of the pyright-internal tests.